### PR TITLE
Ben new branch request

### PR DESCRIPTION
**Pull request by ulrichbenedikt**
Hab das mal versucht, was du vorgeschlagen hast... ist das so richtig ist?

### CODE REVIEW

File: assets/variables.scss [hunk 1]
```diff
@@ -120,15 +120,18 @@ $headings: (
...
-  z-index: 2;
-  height: 100%;
-}
-.v-slide-group__next {
-  right: 0; // -23px;
-}
-.v-slide-group__prev {
-  left: 0; // -23px;
-}
+
```
Comment (fellnerse) [inline assets/variables.scss:123]: remove comment

Comment (fellnerse) [inline assets/variables.scss:123]: remove comment

File: components/Coaching/ContactStatus.vue [hunk 2]
```diff
@@ -246,19 +270,25 @@ export default {
...
           // eslint-disable-next-line vue/no-mutating-props
           this.response.video = v
-          /* this.$stripe.redirectToCheckout({
+          const paymentID = (
+            await this.$fire.functions.httpsCallable('stripe-payCoaching')({
+              responseID: this.response.id,
+              isDev: this.$config.isDev,
+            })
+          ).data
+          this.$stripe.redirectToCheckout({
```
Comment (fellnerse) [inline components/Coaching/ContactStatus.vue:286]: does it work?

File: components/Shared/ServiceOverview.vue [hunk 3]
```diff
@@ -96,7 +96,7 @@
   </v-row>
 </template>
 
-<script>
+<script setup>
```
Comment (fellnerse) [inline components/Shared/ServiceOverview.vue:99]: what does this mean?

File: pages/frauen/index.vue [hunk 4]
```diff
@@ -103,3 +107,19 @@ export default {
   },
 }
 </script>
+
+<style>
```
Comment (fellnerse) [inline pages/frauen/index.vue:111]: i think we should use scoped everywhere where we do not want the css applied to all other components/pages

**Review by fellnerse — CHANGES_REQUESTED** [review]
looks good, some minor changes

File: assets/variables.scss (hunk 1 above)
Comment (fellnerse) [reply assets/variables.scss:123]: this should be the lines below
not the preview shown by github

File: components/Shared/ServiceOverview.vue (hunk 3 above)
Comment (ulrichbenedikt) [reply components/Shared/ServiceOverview.vue:99]: no idea... dont know why it is there...

**Comment by ulrichbenedikt** [comment]
done

**Pull request merged**